migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -166,6 +166,23 @@ In {product-title} 3.11, OpenShift SDN was the default networking plugin in Red
 
 For information on migrating to OVN-Kubernetes from OpenShift SDN, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
 
+[WARNING]
+====
+It is not possible to upgrade a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
+====
+
+For information on OVN-Kubernetes features that are similar to features in the OpenShift SDN plugin see:
```

Comment:
In the 4.16 docs, this is already covered by the migration docs you linked to above, so I'm not sure you need to copy this part

Reply:
Good point, Dan. I'll remove this, but I think we should still keep the warning. What do you think? @danwinship

Reply:
yes, exactly. keep the warning, lose the rest